### PR TITLE
fixed BatchNormalization layer package to reference keras_resnet

### PR DESCRIPTION
**Pull request by A116744**
base keras does not have the option to freeze batch norm layers

### CODE REVIEW

File: keras_resnet/blocks/_time_distributed_2d.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def f(x):
 
         if block == 0:
             shortcut = keras.layers.TimeDistributed(keras.layers.Conv2D(filters * 4, (1, 1), strides=stride, use_bias=False, **parameters), name="res{}{}_branch1".format(stage_char, block_char))(x)
-            shortcut = keras.layers.TimeDistributed(keras.layers.BatchNormalization(axis=axis, epsilon=1e-5, freeze=freeze_bn), name="bn{}{}_branch1".format(stage_char, block_char))(shortcut)
+            shortcut = keras.layers.TimeDistributed(keras_resnet.layers.BatchNormalization(axis=axis, epsilon=1e-5, freeze=freeze_bn), name="bn{}{}_branch1".format(stage_char, block_char))(shortcut)
```
Comment (dvalent1) [inline keras_resnet/blocks/_time_distributed_2d.py]: long function call hurts my eyes, but I am assuming this is something they are ok with?

**Comment by A116744** [comment]
Yeah, that's in their library. Not how I'd write it personally.
